src/main/scala/uk/gov/nationalarchives/notifications/LambdaRunner.scala

```diff
@@ -66,7 +66,31 @@ object LambdaRunner extends App {
        |
        |""".stripMargin
 
-  val inputStream = new ByteArrayInputStream(exportSuccessMessage.getBytes)
+  val transformEngineRetryMessage =
```

Comment:
I don't think you need all the other json here, just `{"Records" : [{ "body" "..."}] }` It does work with it but it might make it easier to parse if there's just the body field.